src/api/app/components/workflow_artifacts_per_step_component.rb

```diff
@@ -34,6 +37,37 @@ def call
 
   private
 
+  def artifacts_for_set_flag(parsed_artifacts)
+    capture do
+      list_of_flags(parsed_artifacts[:flags])
+    end
+  end
+
+  def list_of_flags(flags)
+    flags.each do |flag|
+      concat(flag_step_sentence(flag))
+    end
+  end
+
+  def flag_step_sentence(flag)
+    if flag[:package]
+      path = helpers.package_show_path(project: flag[:project], package: flag[:package])
+      text = "#{flag[:project]}/#{flag[:package]}"
+    else
+      path = helpers.project_show_path(project: flag[:project])
+      text = "#{flag[:project]}"
+    end
+
+    tag.li do
```

Comment:
This will quickly become hard to parse/read once there are a few lines, especially if most of them are for a repository and architecture. At the very least, we should work with text hierarchy to put emphasis on the important information. Something even better would be to have a proper interface for this. It doesn't have to be exactly like what we have in the repository reviews of a project/package, but it is a good inspiration:

![interface](https://github.com/openSUSE/open-build-service/assets/1102934/bae34a8b-a54a-40f0-94cc-fbb61d370579)

Reply:
Do users have a large number of flags in their set_flag step? Because the table makes sense when the data is big. The current solution is easily readable now even with 5 flags. I can bold the tags etc but table seems extra if the information about set flags is not too large

Reply:
I don't have statistics on how many flags users set with the `set_flags` step. I agree that a table is overkill for cases with only a few flags. As long as the text hierarchy is improved, it should be good enough for now.